test/expect/TestScript.test_annot_ast_mypy_fn.expect

```diff
@@ -6,4 +6,4 @@ foo(bool x, (Tensor, Tensor) y) -> ((bool, bool))
 foo(float[3] x, (Tensor, Tensor) y) -> ((float[], float[]))
```

Comment:
Are these `expect` file changes intended? Looks like it is adding empty line at end of each file?